package-testing/testing-api/src/ufc/data.ts

```diff
@@ -0,0 +1,29 @@
+import * as crypto from 'crypto';
+
+const dataFiles: Record<string, { ufc: string; ufcDataVersion: string; bandits: string }> = {};
+const clientDataMap: Record<string, string> = {};
+
+export const getDataForRequest = (sdkName: string) => {
+  // Get the scenario label for this client.
+  let label = clientDataMap[sdkName];
+
+  // If there was no label, assign the first of the data files (or null).
+  if (!label) label = Object.keys(dataFiles)[0] ?? null;
+
+  console.log(`Returning ${label} for ${sdkName}`);
+
+  // If still no label, return null.
+  if (!label) return null;
+
+  return dataFiles[label] ?? null;
```

Comment:
Minor, but I find ternary easier to read than early returns in situations like this
```ts
return label ? dataFiles[label] ? null;
```